src/components/Ipsums.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div>
+        <h2>
+            Add component will be here. 
+        </h2>
+        <IpsumsList
+        :Ipsums="data"
```

Comment:
```suggestion
         :ipsums="data"
        
```
Variable names like this should be lowercase to start.